pkg/api/v1/conversion.go

```diff
@@ -125,7 +125,9 @@ func addConversionFuncs() {
 	err = api.Scheme.AddFieldLabelConversionFunc("v1", "Namespace",
 		func(label, value string) (string, string, error) {
 			switch label {
-			case "status.phase":
+			case "name":
```

Comment:
Even though it's a bug, it's part of an API kept for backwards compatibility as its [comment](https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/namespace/strategy.go#L150) suggests. If we don't define a label conversion for it, it's indeed a bug because although defined, it's unusable. I'd rather add a comment here that `"name"` is deprecated.